tests/xharness/Jenkins.cs

```diff
@@ -1366,8 +1367,9 @@ void GenerateReportImpl (Stream stream, StreamWriter markdown_summary = null)
 					markdown_summary.Write ($"# Test run in progress: ");
 					markdown_summary.Write (string.Join (", ", list));
 					markdown_summary.WriteLine ();
-				} else if (failedTests.Any ()) {
-					markdown_summary.WriteLine ($"{failedTests.Count ()} tests failed, {passedTests.Count ()} tests passed.");
+				} else if (failedTests.Any () || skippedTests.Any ()) {
+					var skipped = skippedTests.Any () ? $" and {skippedTests.Count ()} tests skipped" : "";
+					markdown_summary.WriteLine ($"{failedTests.Count ()} tests failed, {passedTests.Count ()} tests passed{skipped}.");
```

Comment:
The logic does not read well, likely why there's 2 calls to `skippedTests.Any ()`

It looks like what we want is
```
if (fail) { show failures + skipped }
else if (skipped) { show skipped }
else if (pass) { show passed }
...
```

Reply:
not that `Any` followed by `Count` is much better ;-)